pxtlib/docsrender.ts

```diff
@@ -231,7 +231,7 @@ namespace pxt.docs {
         }
 
         params["menu"] = (theme.docMenu || []).map(e => recMenu(e, 0)).join("\n")
-        params["TOC"] = TOC.map(e => recTOC(e, 0)).join("\n")
+        params["TOC"] = TOC.map(e => recTOC(e, 1)).join("\n")
```

Comment:
changes in render require cloud redeploy

Reply:
ok. this applies also for anything living in pxtlib right? I'm working on the lang picker for docs, which will have changes there.

Reply:
render is special because it is also used by our cloud to render doc pages. Otherwise changes in pxtlib mostly impact the client.

Reply:
This means that we need to coordinate upgrading the cloud and merging this PR